Teensy/src/gpio/gpio.cpp

```diff
@@ -60,7 +84,60 @@ static int32_t gpio_getGearPosition()
 
 static uint32_t gpio_getSteeringWheelPosition()
 {
-    return 0;
+    #include <Wire.h>
+    #include <Arduino.h>
+   const int MPU = 0x68; //MPU6050 I2C Address
+   float AccX, AccY, AccZ;
+   float GyroX, GyroY, GyroZ;
+   float gyroAngleX, gyroAngleY, gyroAngleZ;
+   float elapsedTime, currentTime, previousTime;
+   float startPosX, startPosY, startPosZ;
+   float currentPosX, currentPosY, currentPosZ;
+   float yaw;
+   float calibrationvalue;
+   int c = 0;
+
+
+    Serial.begin(19200);
+    Wire.begin();
+    Wire.beginTransmission(MPU);
+    Wire.write(0x6B);
+    Wire.write(0x00);
+    Wire.endTransmission(true);
+    delay(20);
```

Comment:
This initialisation of the I2C communication for the steering wheel should be separate so its not repeated each time we want to get the steering wheel position.

Create a function in `gpio.cpp` called `gpio_init()`, which can then be called from `main.cpp`'s `setup()`. This is where you should do the functions you've described above.